docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,416 @@
+space.
+
+In order to reuse existing (already verified) fastd-keypairs for
+WireGuard, a key derivation procedure is `currently being
+developed <https://github.com/freifunk-gluon/gluon/pull/2601>`__. This
+should ease migration from fastd to WireGuard in case whitelisting VPN
+keys is desired.
+
+fastd L2TP
+~~~~~~~~~~
+
+fastd can now act as a connection broker for unencrypted L2TP-based
+tunneling within Gluons mesh-vpn framework. This new ``null@l2tp``
+connection Method allows for increased performance within existing
+fastd setups.
+
+In addition to a sufficiently
+:ref:`configured fastd-based VPN server<vpn-gateway-configuration>`,
+this requires further modifications to a site :ref:`VPN fastd methods`.
+
+Major changes
+-------------
+
+OpenWrt
+~~~~~~~
+
+This release is based on the newest OpenWrt 22.03 release branch.
+It ships with Linux kernel 5.10 as well as wireless-backports 5.15.
+
+
+Network changes (DSA / Upgrade-Behavior)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``ramips-mt7621`` and ``lantiq-xrx200`` targets now use the upstream DSA
+subsystem instead of OpenWrt swconfig for managing ethernet switches.
+
+Gluon detects the existing user-intent and automatically applies it over
+to DSA syntax. See the section about network reconfiguration for more
+details.
+
+System reconfiguration
+~~~~~~~~~~~~~~~~~~~~~~
+
+The network and system-LED configurations are now re-generated after
+each update / invocation of ``gluon-reconfigure``.
+
+The user-intent is preserved within Gluon’s implemented functionality
+(Wired-Mesh / Client access / WAN).
+
+As an additional feature, Gluon now supports assigning roles to
+interfaces. This behavior is explained
+:ref:`here<wired-mesh-commandline>`.
+
+Site changes
+------------
+
+VPN provider MTU
+~~~~~~~~~~~~~~~~
+
+To account for multiple VPN methods available for a site, the MTU used
+for the VPN tunnel connection is now moved to the specific VPN provider
+configuration. For fastd this means that ``mesh_vpn.mtu`` needs to be
+moved to ``mesh_vpn.fastd.mtu``. (`#2352 <https://github.com/freifunk-gluon/gluon/pull/2352>`__)
+
+Preconfigured Interfaces Roles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of ``mesh_on_wan`` and ``mesh_on_lan`` there is now a
+``interfaces`` block to configure the default behavior of network
+interfaces. Details can be found in the 
+:ref:`documentation<user-site-interfaces>`.
+
+Minor changes
+-------------
+
+-  The ``brcm2708-bcm2708`` ``brcm2708-bcm2709`` ``brcm2708-bcm2710``
+   targets were renamed to ``bcm27xx-bcm2708`` ``bcm27xx-bcm2709`` and
+   ``bcm27xx-bcm2710``
+-  The GL.iNet GL-AR750S was moved to the ``ath79-nand`` subtarget
+-  Gluon now ships the ath10k-ct firmware derivation for
+   QCA9886 / QCA9888 / QCA9896 / QCA9898 / QCA9984 /
+   QCA9994 / IPQ4018 / IPQ4028 / IPQ4019 / IPQ4029
+   radios (`#2541 <https://github.com/freifunk-gluon/gluon/pull/2541>`__)
+-  WolfSSL instead of OpenSSL is now used when built with WPA3 support.
```

Comment:
```suggestion
-  WolfSSL instead of OpenSSL is now used when built with WPA3 support
```

the other bulletpoints in this section do not have dots either